dogstatsd.py

```diff
@@ -336,11 +339,12 @@ class Server(object):
     """
     A statsd udp server.
     """
-    def __init__(self, metrics_aggregator, host, port, forward_to_host=None, forward_to_port=None):
+    def __init__(self, metrics_aggregator, host, port, forward_to_host=None, forward_to_port=None,
+                 buffer_size=DEFAULT_BUFFER_SIZE):
```

Comment:
Here you have to decide if buffer_size is in B or KB. DEFAULT_BUFFER_SIZE is in KB, but you're passing the buffer size in BYTES.

You should be good if you set `buffer_size=DEFAULT_BUFFER_SIZE * 1024`, although it looks kind of ugly. You could also define another constant `DEFAULT_BUFFER_SIZE_BYTES = DEFAULT_BUFFER_SIZE * 1024`, and then set parameter `buffer_size=DEFAULT_BUFFER_SIZE_BYTES`. I prefer the latter approach.